readme.md

```diff
@@ -725,3 +725,9 @@ What's the best tool for automatically ignoring long email threads about tech bu
 **A:** They retreated back into their shell
 
 ---
+
+**Q:** What did the egg say when he got to drunk?
+
+**A:** Omelette!
```

Comment:
I'm not sure I understand.

Perhaps it should be "What did the egg say when it got too close to the fire?"

Or maybe I don't understand? 🤷‍♂

Reply:
Ome-lette
om-let
I'm-lit

It might come down to the enunciation of the word omelette.